proto/public/uber/cadence/api/v1/error.proto

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2020 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+syntax = "proto3";
+
+package uber.cadence.api.v1;
+
+option go_package = "apiv1";
+option java_multiple_files = true;
+option java_outer_classname = "ApiProto";
+option java_package = "com.uber.cadence.api.v1";
+
+message WorkflowExecutionAlreadyStartedError {
+  string start_request_id = 1;
+  string run_id = 2;
+}
+
+message EntityNotExistsError {
+  string current_cluster = 1;
+  string active_cluster = 2;
+}
+
+message DomainNotActiveError {
+  string domain = 1;
+  string current_cluster = 2;
+  string active_cluster = 3;
+}
+
+message RetryTaskV2Error {
+  string domain_id = 1;
```

Comment:
though this is in multiple places, I just noticed it here:
there's some inconsistency between `domain` and `domain_id`.  do they ever mean different things?  or can they be standardized?

Reply:
Tes. domain and domain_id are different. Domain is the actual user domain name. Domain ID is a UUID.

Reply:
after a closer look: it kinda looks like `domain_id` is used only for internal / worker purposes?  that's probably fine then / makes sense - uuids are easier for identity.

there is one exception though: `WorkflowExecutionInfo` is user-visible and only contains `domain_id`, not `domain` (unless it's nested in one of the other fields that I haven't noticed?).  worth adding `domain` there maybe?